### DISCUSSION TITLE
recommendations

### DISCUSSION THREAD

**Discussion by abdelmounim-dev** [DBChoco/Muezzin#21] [Ideas]
Assalam alaykum, 
please add the option to stop the adhan without closing the app, and an other feature that would be great is custom reminders, like 15 minutes before every adhan or a specific one, that would be very helpful and thank you, Jazaka Allah Khayran

**Comment by DBChoco** [comment]
Wa Alaykoum Assalam brother,

1. You can already stop the adhan midway by clicking on the stop button on the bottom-left corner: 
![image](https://user-images.githubusercontent.com/44785461/227793348-503afaf0-b824-4954-a12e-f079d9d54df6.png)

2. As for the reminders, it's a very good idea, I'll keep it in mind inshaAllah. For now, you can use the adjustments to bring the Adhan forward by 10 or 15 minutes, that way it'll remind you to go to prepare yourself. InshaAllah if I ever get the time, I'll try and see if I can add a reminder system to the app.

Jazzaka Lahu khairan for the input, may Allah bless you.
Assalam Alaykoum wa Rahmatou Lahi wa Baraketu

**Reply by abdelmounim-dev** [reply]
Thank you, BTW can I talk to you privately  my brother?

**Reply by DBChoco** [reply]
Sure brother, here is my email: dbchoco.muezzin@gmail.com

**Reply by abdelmounim-dev** [reply]
I remembered where the issue was with stopping the adhan, if I close the app but it stays minimized, the adhan will play, and then I can stop it only by quitting the app, Jazaka Allahu khayran and Ramadan Kareem

**Reply by DBChoco** [reply]
Yes that's true, if it's minimized you have to open the app again then click on the "Stop" button, or you can leave the app altogether.

**Comment by martinwprior** [comment]
As I live about 20 mins from the Masjid it would be nice to have a reminder in advance of the adhan as  suggested. In the meantime I note your workaround.

**Comment by DBChoco** [comment]
Could you try this https://github.com/DBChoco/Muezzin/commit/094f052ad6a4db9bf76049a24f5788db0aea1067 little update ?
I didn't release a new version yet, I fear there are still a few bugs that I didn't notice, and I still have to put the translation in. Plus some visual changes; but you should be able to receive a reminder from 1 to 59 minutes before the Adhan now.
If you need help to build it from source, you can ask me here: dbchoco.muezzin@gmail.com

Assalam Alaykoum.

**Comment by hammadrauf** [comment]
Assalam o Alaikum,
This is a great application software. Is it possible to Install and execute this software as a System User in Windows or as Root in Linux. I want to keep it executing even if no one is logged in.
Thank You

Hammad Rauf

**Reply by DBChoco** [reply]
Wa alaykum Assalam wa Rahmatou Lahi wa Baraketu brother Hammad;
Sorry I don't know of a way to do that; perhaps you can find help [here](https://stackoverflow.com/questions/12491364/how-to-run-application-exe-before-windows-startup) ?

Assalam Alaykoum

**Comment by ibrahimnganga** [comment]
Assalam Alaykoum. Thank you for this app its very helpful to me as a new Muslim. I am requesting for addition of a way I can view and learn duas as I go by. Thank you.

**Reply by DBChoco** [reply]
Assalam Alaykoum;
I've already though about this, but I soon hit a roadblock: It is a lot of work to translate all the du'a to 10+ languages, or find them in their respective languages.
And I couldn't find a compiled source that would allow me to easily list all needed du'a.

Here you can find quite a few good du'as: https://sunnah.com/hisn
I already requested an API key from sunnah.com, like 1 year ago, but I still didn't get an answer, so for now I'd recommend using their website directly.

Sorry about that.
Assalam Alaykoum wa Rahmatou Lahi wa Baraketu

**Reply by ibrahimnganga** [reply]
Thank you for the link

**Comment by Gameonfunnn** [comment]
Alsalmu Alaikum, can you please add a widget so i can put it on my desktop in macOS, thanks. And Mashallah what a wonderful app, saved my life by not having to open my phone only to view prayer times :) Allah yubarak feek 🤲
